src/main/java/com/commercetools/project/sync/taxcategory/TaxCategorySyncer.java

```diff
@@ -105,6 +106,7 @@ private static TaxCategoryDraft convertTaxCategoryToTaxCategoryDraft(
   @Nonnull
   private static List<TaxRateDraft> convertTaxRateToTaxRateDraft(
       @Nonnull final List<TaxRate> taxRates) {
+
```

Comment:
I found there is no caffine key-id cache used in TaxCategorySyncer. Is it not necessary?

Reply:
`TaxCategorySyncer` doesn't have any references. So we don't have expansion. Hence no cache is used.